shub/plugins/pgp/models.py

```diff
@@ -48,7 +48,8 @@ def class_prepared(self, sender, **kwargs):
         post_save.connect(self.post_save, sender=self.model)
         post_delete.connect(self.post_delete, sender=self.model)
 
-    def post_save(self, sender, instance, created, **kwargs):
+    @staticmethod
```

Comment:
I don't think we should change these either, these follow the suggested way to implement these callbacks.